### ISSUE TITLE
Maintenance status

### ISSUE THREAD

**Issue by djc** [hickory-dns/resolv-conf#36]
There seems to be a little activity here, which is causing some issues:

- https://github.com/hickory-dns/hickory-dns/issues/1131
- https://github.com/getsentry/relay/issues/2147

@tailhook would you be willing to grant me access to this repo/crates.io so that I can help move this crate forward?

**Comment by hirschenberger** [comment]
Seems to be abandoned and a candidate for a fork.

**Comment by djc** [comment]
I sent an personal email to the maintainer today to see if I can recover the crates.io name.

**Comment by djc** [comment]
Nope. Just reached out to the crates.io team to see if they might transfer ownership (but I guess they won't).

**Comment by tailhook** [comment]
Sorry, I'm mostly off the github these days, unfortunately :(

I've invited @djc both here on github and on crates.io.

**Comment by tailhook** [comment]
Perhaps we can also move this repo to the org, as my situation will unlikely to change in the near future.

**Comment by djc** [comment]
No problem at all, glad to see you're still able to respond, and thanks for granting me access!

> Perhaps we can also move this repo to the org, as my situation will unlikely to change in the near future.

Sounds good! I've temporarily made you a member of the hickory-dns org, I think that should allow you to transfer this repo into it? (I might be able to do it if you're able to give me admin-level access to this repo, but not sure that is possible for repos under a personal account.)

**Comment by djc** [comment]
Going to close this -- feel free to ping me on issues/PRs that could use followup. (Have merged #35.)
